Plenty of emojis, new symbols, kanjis, and improved diagonals

PhMajerus opened the pull request:
This PR contains 304 Emojis, 25 kanjis (following @bluetoad07 new Kanji designs), 30 symbols, and 1 Nerd Font logo.
11 are changes to existing code points to make them consistent with the new ones.
It also improves the 3 existing box drawings diagonals to make them combine into proper straight lines.
Finally, it also adjusts block elements mosaics and related glyphs to fit a unified grid.

Changed:
≈ `U+2248` Almost Equal To (as per issue #154)
≉ `U+2249` Not Almost Equal To (as per issue #154)
⏨ `U+23E8` Decimal Exponent Symbol (as per issue #149)
╱ `U+2571` Box Drawings Light Diagonal Upper Right to Lower Left (as per issue #160)
╲ `U+2572` Box Drawings Light Diagonal Upper Left to Lower Right (as per issue #160)
╳ `U+2573` Box Drawings Light Diagonal Cross (as per issue #160)
☃ `U+2603` Snowman (changed from single-cell to Emoji style as expected by most terminals)
☕ `U+2615` Hot Beverage (changed from single-cell to Emoji style as expected by most terminals)
⭐ `U+2B50` White Medium Star (changed from single-cell to Emoji style as expected by most terminals)
🌱 `U+1F331` Seedling (changed from single-cell to Emoji style as expected by most terminals)
🌳 `U+1F333` Deciduous Tree (changed from single-cell to Emoji style as expected by most terminals)
🍹 `U+1F379` Tropical Drink (changed for consistency with other drinks emojis)
💎 `U+1F48E` Gem Stone (changed from single-cell to Emoji style as expected by most terminals)
🔧 `U+1F527` Wrench (slightly resized to better fit Emojis bounding-box)

Added:
☂ `U+2602` Umbrella (Emoji style as expected by most terminals)
☄ `U+2604` Comet (Emoji style)
☔ `U+2614` Umbrella with Rain Drops
♻ `U+267B` Black Universal Recycling Symbol (Emoji style)
⚓ `U+2693` Anchor
⚽ `U+26BD` Soccer Ball
⚾ `U+26BE` Baseball
⛄ `U+26C4` Snowman Without Snow
⛩ `U+26E9` Shinto Shrine (Emoji style)
⛪ `U+26EA` Church
✻ `U+273B` Teardrop-Spoked Asterisk (as full-width) (as requested in issue #170)
✼ `U+273C` Open Centre Teardrop-Spoked Asterisk (as full-width) (as requested in issue #170)
✽ `U+273D` Heavy Teardrop-Spoked Asterisk (as full-width) (as requested in issue #170)
✾ `U+273E` Six Petalled Black and White Florette (as full-width)
✿ `U+273F` Black Florette (as full-width)
❀ `U+2740` White Florette (as full-width)
❇ `U+2747` Sparkle (as full-width)
❈ `U+2748` Heavy Sparkle (as full-width)
❉ `U+2749` Balloon-Spoked Asterisk (as full-width)
❊ `U+274A` Eight Teardrop-Spoked Propeller Asterisk (as full-width)
❋ `U+274B` Heavy Eight Teardrop-Spoked Propeller Asterisk (as full-width)
⭕ `U+2B55` Heavy Large Circle
伝 `U+4F1D` CJK Unified Ideograph-4F1D
剣 `U+5263` CJK Unified Ideograph-5263
影 `U+5F71` CJK Unified Ideograph-5F71
忍 `U+5FCD` CJK Unified Ideograph-5FCD
愛 `U+611B` CJK Unified Ideograph-611B
戦 `U+6226` CJK Unified Ideograph-6226
斧 `U+65A7` CJK Unified Ideograph-65A7
極 `U+6975` CJK Unified Ideograph-6975
正 `U+6B63` CJK Unified Ideograph-6B63
獣 `U+7363` CJK Unified Ideograph-7363
王 `U+738B` CJK Unified Ideograph-738B
界 `U+754C` CJK Unified Ideograph-754C
罠 `U+7F60` CJK Unified Ideograph-7F60
者 `U+8005` CJK Unified Ideograph-8005
舞 `U+821E` CJK Unified Ideograph-821E
記 `U+8A18` CJK Unified Ideograph-8A18
説 `U+8AAC` CJK Unified Ideograph-8AAC
超 `U+8D85` CJK Unified Ideograph-8D85
魔 `U+9B54` CJK Unified Ideograph-9B54
龍 `U+9F8D` CJK Unified Ideograph-9F8D
 `U+F322` PUA - EndeavourOS (nf-linux-endeavour, as per issue #137)
𝍲 `U+1D372` Ideographic Tally Mark One (based on 正 `U+6B63`)
𝍳 `U+1D373` Ideographic Tally Mark Two (based on 正 `U+6B63`)
𝍴 `U+1D374` Ideographic Tally Mark Three (based on 正 `U+6B63`)
𝍵 `U+1D375` Ideographic Tally Mark Four (based on 正 `U+6B63`)
𝍶 `U+1D376` Ideographic Tally Mark Five (identical to 正 `U+6B63`)
🄎 `U+1F10E` Circled Anticlockwise Arrow (as full-width)
🄏 `U+1F10F` Circled Dollar Sign With Overlaid Backslash (as full-width)
🄪 `U+1F12A` Tortoise Shell Bracketed Latin Capital Letter S (as full-width)
🄫 `U+1F12B` Circled Italic Latin Capital Letter C (as full-width)
🄬 `U+1F12C` Circled Italic Latin Capital Letter R (as full-width)
🄭 `U+1F12D` Circled Cd (as full-width)
🄮 `U+1F12E` Circled Wz (as full-width)
🄯 `U+1F12F` Copyleft Symbol (as full-width)
🅪 `U+1F16A` Raised MC Sign (based on existing raised TM sign `U+2122`)
🅫 `U+1F16B` Raised MD Sign
🅬 `U+1F16C` Raised MR Sign
🅭 `U+1F16D` Circled CC (as full-width)
🅮 `U+1F16E` Circled C With Overlaid Backslash (as full-width)
🅯 `U+1F16F` Circled Human Figure (as full-width)
🌍 `U+1F30D` Earth Globe Europe-Africa
🌎 `U+1F30E` Earth Globe Americas
🌏 `U+1F30F` Earth Globe Asia-Australia
🌐 `U+1F310` Globe With Meridians
🌙 `U+1F319` Crescent Moon (sized like other moon emojis)
🌟 `U+1F31F` Glowing Star (Emoji style)
🌠 `U+1F320` Shooting Star
🌭 `U+1F32D` Hot Dog
🌮 `U+1F32E` Taco
🌯 `U+1F32F` Burrito
🌰 `U+1F330` Chestnut
🌲 `U+1F332` Evergreen Tree
🌴 `U+1F334` Palm Tree
🌵 `U+1F335` Cactus
🌶 `U+1F336` Hot Pepper (as full-width)
🌷 `U+1F337` Tulip
🌹 `U+1F339` Rose
🌺 `U+1F33A` Hibiscus
🌻 `U+1F33B` Sunflower
🌼 `U+1F33C` Blossom
🌽 `U+1F33D` Ear of Maize
🌾 `U+1F33E` Ear of Rice
🌿 `U+1F33F` Herb
🍁 `U+1F341` Maple Leaf
🍂 `U+1F342` Fallen Leaf
🍃 `U+1F343` Leaf Fluttering In Wind
🍄 `U+1F344` Mushroom
🍅 `U+1F345` Tomato
🍆 `U+1F346` Aubergine
🍇 `U+1F347` Grapes
🍈 `U+1F348` Melon
🍉 `U+1F349` Watermelon
🍊 `U+1F34A` Tangerine
🍋 `U+1F34B` Lemon
🍌 `U+1F34C` Banana
🍍 `U+1F34D` Pineapple
🍐 `U+1F350` Pear
🍑 `U+1F351` Peach
🍒 `U+1F352` Cherries
🍓 `U+1F353` Strawberry
🍔 `U+1F354` Hamburger
🍕 `U+1F355` Slice of Pizza
🍖 `U+1F356` Meat On Bone
🍗 `U+1F357` Poultry Leg
🍘 `U+1F358` Rice Cracker
🍙 `U+1F359` Rice Ball
🍚 `U+1F35A` Cooked Rice
🍛 `U+1F35B` Curry and Rice
🍜 `U+1F35C` Steaming Bowl
🍝 `U+1F35D` Spaghetti
🍞 `U+1F35E` Bread
🍟 `U+1F35F` French Fries
🍠 `U+1F360` Roasted Sweet Potato
🍡 `U+1F361` Dango
🍢 `U+1F362` Oden
🍣 `U+1F363` Sushi
🍤 `U+1F364` Fried Shrimp
🍥 `U+1F365` Fish Cake with Swirl Design
🍦 `U+1F366` Soft Ice Cream
🍧 `U+1F367` Shaved Ice
🍨 `U+1F368` Ice Cream
🍩 `U+1F369` Doughnut
🍪 `U+1F36A` Cookie
🍫 `U+1F36B` Chocolate Bar
🍬 `U+1F36C` Candy
🍭 `U+1F36D` Lollipop
🍮 `U+1F36E` Custard
🍯 `U+1F36F` Honey Pot
🍰 `U+1F370` Shortcake
🍱 `U+1F371` Bento Box
🍲 `U+1F372` Pot of Food
🍳 `U+1F373` Cooking
🍴 `U+1F374` Fork and Knife
🍵 `U+1F375` Teacup Without Handle
🍶 `U+1F376` Sake Bottle and Cup
🍷 `U+1F377` Wine Glass
🍸 `U+1F378` Cocktail Glass
🍺 `U+1F37A` Beer Mug
🍻 `U+1F37B` Clinking Beer Mugs
🍼 `U+1F37C` Baby Bottle
🍽 `U+1F37D` Fork and Knife with Plate (as full-width)
🍾 `U+1F37E` Bottle with Popping Cork
🍿 `U+1F37F` Popcorn
🎀 `U+1F380` Ribbon
🎂 `U+1F382` Birthday Cake
🎃 `U+1F383` Jack-O-Lantern
🎄 `U+1F384` Christmas Tree
🎅 `U+1F385` Father Christmas
🎈 `U+1F388` Balloon
🎉 `U+1F389` Party Popper
🎊 `U+1F38A` Confetti Ball
🎋 `U+1F38B` Tanabata Tree
🎌 `U+1F38C` Crossed Flags
🎍 `U+1F38D` Pine Decoration
🎎 `U+1F38E` Japanese Dolls
🎏 `U+1F38F` Carp Streamer
🎐 `U+1F390` Wind Chime
🎑 `U+1F391` Moon Viewing Ceremony
🎒 `U+1F392` School Satchel
🎓 `U+1F393` Graduation Cap
🎩 `U+1F3A9` Top Hat
🎪 `U+1F3AA` Circus Tent
🎬 `U+1F3AC` Clapper Board
🎯 `U+1F3AF` Direct Hit
🎰 `U+1F3B0` Slot Machine
🎲 `U+1F3B2` Game Die
🎳 `U+1F3B3` Bowling
🎴 `U+1F3B4` Flower Playing Cards
🎵 `U+1F3B5` Musical Note
🎶 `U+1F3B6` Multiple Musical Notes
🎹 `U+1F3B9` Musical Keyboard
🎼 `U+1F3BC` Musical Score
🏀 `U+1F3C0` Basketball and Hoop
🏅 `U+1F3C5` Sports Medal
🏆 `U+1F3C6` Trophy
🏈 `U+1F3C8` American Football
🏉 `U+1F3C9` Rugby Football
🏐 `U+1F3D0` Volleyball
🏙 `U+1F3D9` Cityscape (Emoji style)
🏚 `U+1F3DA` Derelict House Building (Emoji style)
🏛 `U+1F3DB` Classical Building (Emoji style)
🏠 `U+1F3E0` House Building
🏡 `U+1F3E1` House With Garden
🏢 `U+1F3E2` Office Building
🏣 `U+1F3E3` Japanese Post Office
🏤 `U+1F3E4` European Post Office
🏥 `U+1F3E5` Hospital
🏦 `U+1F3E6` Bank
🏧 `U+1F3E7` Automated Teller Machine
🏨 `U+1F3E8` Hotel
🏩 `U+1F3E9` Love Hotel
🏪 `U+1F3EA` Convenience Store
🏫 `U+1F3EB` School
🏬 `U+1F3EC` Department Store
🏭 `U+1F3ED` Factory
🏮 `U+1F3EE` Izakaya Lantern
🏯 `U+1F3EF` Japanese Castle
🏰 `U+1F3F0` European Castle
🐓 `U+1F413` Rooster
🐟 `U+1F41F` Fish
🐠 `U+1F420` Tropical Fish
🐡 `U+1F421` Blowfish
🐣 `U+1F423` Hatching Chick
🐤 `U+1F424` Baby Chick
🐥 `U+1F425` Front-Facing Baby Chick
👀 `U+1F440` Eyes
👁 `U+1F441` Eye (Emoji style)
👄 `U+1F444` Mouth
💈 `U+1F488` Barber Pole
💊 `U+1F48A` Pill
💋 `U+1F48B` Kiss Mark
💍 `U+1F48D` Ring
💏 `U+1F48F` Kiss
💐 `U+1F490` Bouquet
💑 `U+1F491` Couple with Heart
💒 `U+1F492` Wedding
💟 `U+1F49F` Heart Decoration
💢 `U+1F4A2` Anger Symbol
💦 `U+1F4A6` Splashing Sweat Symbol
💧 `U+1F4A7` Droplet
💨 `U+1F4A8` Dash Symbol
💱 `U+1F4B1` Currency Exchange
💳 `U+1F4B3` Credit Card
💴 `U+1F4B4` Banknote With Yen Sign
💵 `U+1F4B5` Banknote With Dollar Sign
💶 `U+1F4B6` Banknote With Euro Sign
💷 `U+1F4B7` Banknote With Pound Sign
💸 `U+1F4B8` Money with Wings
💹 `U+1F4B9` Chart with Upwards Trend and Yen Sign
📈 `U+1F4C8` Chart with Upwards Trend
📉 `U+1F4C9` Chart with Downwards Trend
📊 `U+1F4CA` Bar Chart
📍 `U+1F4CD` Round Pushpin
📎 `U+1F4CE` Paperclip
📛 `U+1F4DB` Name Badge
📟 `U+1F4DF` Pager
📱 `U+1F4F1` Mobile Phone
📲 `U+1F4F2` Mobile Phone With Rightwards Arrow At Left
📹 `U+1F4F9` Video Camera
📺 `U+1F4FA` Television
📻 `U+1F4FB` Radio
📼 `U+1F4FC` Videocassette
🔌 `U+1F50C` Electric Plug
🔍 `U+1F50D` Left-Pointing Magnifying Glass
🔎 `U+1F50E` Right-Pointing Magnifying Glass
🔏 `U+1F50F` Lock with Ink Pen
🔔 `U+1F514` Bell (Emoji style)
🔕 `U+1F515` Bell with Cancellation Stroke
🔦 `U+1F526` Electric Torch
🔨 `U+1F528` Hammer
🔩 `U+1F529` Nut and Bolt
🔰 `U+1F530` Japanese Symbol For Beginner
🔱 `U+1F531` Trident Emblem
🕹 `U+1F579` Joystick (Emoji style)
🕾 `U+1F57E` White Touchtone Telephone
🕿 `U+1F57F` Black Touchtone Telephone
🖄 `U+1F584` Envelope with Lightning
🖅 `U+1F585` Flying Envelope
🖆 `U+1F586` Pen Over Stamped Envelope
🗄 `U+1F5C4` File Cabinet (Emoji style)
🗻 `U+1F5FB` Mount Fuji
🗼 `U+1F5FC` Tokyo Tower
🗾 `U+1F5FE` Silhouette of Japan
😍 `U+1F60D` Smiling Face with Heart-Shaped Eyes
🚥 `U+1F6A5` Horizontal Traffic Light
🚦 `U+1F6A6` Vertical Traffic Light
🚧 `U+1F6A7` Construction Sign
🚨 `U+1F6A8` Police Cars Revolving Light
🚩 `U+1F6A9` Triangular Flag On Post
🚪 `U+1F6AA` Door
🚫 `U+1F6AB` No Entry Sign
🛑 `U+1F6D1` Octagonal Sign
🛕 `U+1F6D5` Hindu Temple
🛖 `U+1F6D6` Hut
🛫 `U+1F6EB` Airplane Departure
🛬 `U+1F6EC` Airplane Arriving
🤩 `U+1F929` Grinning Face with Star Eyes
🥀 `U+1F940` Wilted Flower
🥁 `U+1F941` Drum with Drumsticks
🥇 `U+1F947` First Place Medal
🥈 `U+1F948` Second Place Medal
🥉 `U+1F949` Third Place Medal
🥎 `U+1F94E` Softball
🥐 `U+1F950` Croissant
🥑 `U+1F951` Avocado
🥒 `U+1F952` Cucumber
🥔 `U+1F954` Potato
🥕 `U+1F955` Carrot
🥖 `U+1F956` Baguette Bread
🥙 `U+1F959` Stuffed Flatbread
🥚 `U+1F95A` Egg
🥛 `U+1F95B` Glass of Milk
🥜 `U+1F95C` Peanuts
🥝 `U+1F95D` Kiwifruit
🥞 `U+1F95E` Pancakes
🥟 `U+1F95F` Dumpling
🥠 `U+1F960` Fortune Cookie
🥤 `U+1F964` Cup with Straw
🥮 `U+1F96E` Moon Cake
🥯 `U+1F96F` Bagel
🥷 `U+1F977` Ninja
🦋 `U+1F98B` Butterfly
🧉 `U+1F9C9` Mate Drink
🧊 `U+1F9CA` Ice Cube
🧋 `U+1F9CB` Bubble Tea
🧨 `U+1F9E8` Firecracker
🧩 `U+1F9E9` Jigsaw Puzzle Piece
🧭 `U+1F9ED` Compass
🧱 `U+1F9F1` Brick
🧲 `U+1F9F2` Magnet
🧵 `U+1F9F5` Spool of Thread
🧶 `U+1F9F6` Ball of Yarn
🧷 `U+1F9F7` Safety Pin
🧸 `U+1F9F8` Teddy Bear
🧺 `U+1F9FA` Basket
🧻 `U+1F9FB` Roll of Paper
🧼 `U+1F9FC` Bar of Soap
🧽 `U+1F9FD` Sponge
🩸 `U+1FA78` Drop of Blood
🩹 `U+1FA79` Adhesive Bandage
🪎 `U+1FA8E` Treasure Chest (Unicode 17.0)
🪏 `U+1FA8F` Shovel
🪙 `U+1FA99` Coin
🪚 `U+1FA9A` Carpentry Saw
🪛 `U+1FA9B` Screwdriver
🪠 `U+1FAA0` Plunger
🪪 `U+1FAAA` Identification Card
🪾 `U+1FABE` Leafless Tree
🫀 `U+1FAC0` Anatomical Heart
🫁 `U+1FAC1` Lungs
🫆 `U+1FAC6` Fingerprint
🫐 `U+1FAD0` Blueberries
🫑 `U+1FAD1` Bell Pepper
🫒 `U+1FAD2` Olive
🫓 `U+1FAD3` Flatbread
🫔 `U+1FAD4` Tamale
🫕 `U+1FAD5` Fondue
🫖 `U+1FAD6` Teapot
🫗 `U+1FAD7` Pouring Liquid
🫘 `U+1FAD8` Beans
🫙 `U+1FAD9` Jar
🫚 `U+1FADA` Ginger
🫛 `U+1FADB` Pea Pod
🫜 `U+1FADC` Root Vegetable
🫦 `U+1FAE6` Biting Lip
🫧 `U+1FAE7` Bubbles
🯺 `U+1FBFA` Alarm Bell Symbol (Symbols for Legacy Computing Unicode 17.0)

Here's an example of most of the above (all except 33 emojis from last commit):
<img width="105" height="589" alt="image" src="https://github.com/user-attachments/assets/40c2700d-7818-4f37-bc4a-cbc58782fbdf" />

### Fixes for block elements and eights

This PR also contains fixes to unify the grid used for 8×8 mosaics and related, as described in #173.
I'm not sure which ones I had to modify or not, but it's a review of the ranges `U+2580` to `U+259F`, `U+1FB70` to `U+1FB7B`, `U+1FB82` to `U+1FB8B`, `U+1CEA5` to `U+1CEA7`, and `U+1CEAC` to `U+1CEAE`.
This also required adjusting some other TUI elements using the 8×8 grid such as MouseText in `U+1FB7C` to `U+1FB81`, `U+1FBB5` to `U+1FBB8`, `U+1FBBC`, `U+1CC8E`, and `U+1CC8F`.

Before:
<img width="910" height="591" alt="image" src="https://github.com/user-attachments/assets/96d726c7-6e4c-4110-97ef-91823a49e437" />

After:
<img width="910" height="591" alt="image" src="https://github.com/user-attachments/assets/88bdd32e-ca18-45ad-b5cd-bdb3cab4a237" />

It looks almost identical, but it makes a big difference for ASCII-art and other pseudo-pixels-based images in terminals.

PhMajerus committed:
264 Emojis, 22 kanjis, 27 symbols, and 1 Nerd Font logo

Changed:
⏨ U+23E8 Decimal Exponent Symbol (as per issue #149)
☃ U+2603 Snowman (changed from single-cell to Emoji style as expected by most terminals)
☕ U+2615 Hot Beverage (changed from single-cell to Emoji style as expected by most terminals)
⭐ U+2B50 White Medium Star (changed from single-cell to Emoji style as expected by most terminals)
🌱 U+1F331 Seedling (changed from single-cell to Emoji style as expected by most terminals)
🌳 U+1F333 Deciduous Tree (changed from single-cell to Emoji style as expected by most terminals)
🍹 U+1F379 Tropical Drink (changed for consistency with other drinks emojis)
💎 U+1F48E Gem Stone (changed from single-cell to Emoji style as expected by most terminals)
🔧 U+1F527 Wrench (slightly resized to better fit Emojis bounding-box)
Added:
☂ U+2602 Umbrella (Emoji style as expected by most terminals)
☄ U+2604 Comet (Emoji style)
☔ U+2614 Umbrella with Rain Drops
♻ U+267B Black Universal Recycling Symbol (Emoji style)
⚓ U+2693 Anchor
⚽ U+26BD Soccer Ball
⚾ U+26BE Baseball
⛄ U+26C4 Snowman Without Snow
⛩ U+26E9 Shinto Shrine (Emoji style)
⛪ U+26EA Church
✻ U+273B Teardrop-Spoked Asterisk (as full-width)
✼ U+273C Open Centre Teardrop-Spoked Asterisk (as full-width)
✽ U+273D Heavy Teardrop-Spoked Asterisk (as full-width)
✾ U+273E Six Petalled Black and White Florette (as full-width)
✿ U+273F Black Florette (as full-width)
❀ U+2740 White Florette (as full-width)
❇ U+2747 Sparkle (as full-width)
❈ U+2748 Heavy Sparkle (as full-width)
❉ U+2749 Balloon-Spoked Asterisk (as full-width)
❊ U+274A Eight Teardrop-Spoked Propeller Asterisk (as full-width)
❋ U+274B Heavy Eight Teardrop-Spoked Propeller Asterisk (as full-width)
⭕ U+2B55 Heavy Large Circle
伝 U+4F1D CJK Unified Ideograph-4F1D
影 U+5F71 CJK Unified Ideograph-5F71
忍 U+5FCD CJK Unified Ideograph-5FCD
愛 U+611B CJK Unified Ideograph-611B
戦 U+6226 CJK Unified Ideograph-6226
斧 U+65A7 CJK Unified Ideograph-65A7
極 U+6975 CJK Unified Ideograph-6975
正 U+6B63 CJK Unified Ideograph-6B63
獣 U+7363 CJK Unified Ideograph-7363
王 U+738B CJK Unified Ideograph-738B
界 U+754C CJK Unified Ideograph-754C
罠 U+7F60 CJK Unified Ideograph-7F60
舞 U+821E CJK Unified Ideograph-821E
記 U+8A18 CJK Unified Ideograph-8A18
説 U+8AAC CJK Unified Ideograph-8AAC
超 U+8D85 CJK Unified Ideograph-8D85
魔 U+9B54 CJK Unified Ideograph-9B54
 U+F322 PUA - EndeavourOS (nf-linux-endeavour, as per issue #137)
𝍲 U+1D372 Ideographic Tally Mark One (based on 正 U+6B63)
𝍳 U+1D373 Ideographic Tally Mark Two (based on 正 U+6B63)
𝍴 U+1D374 Ideographic Tally Mark Three (based on 正 U+6B63)
𝍵 U+1D375 Ideographic Tally Mark Four (based on 正 U+6B63)
𝍶 U+1D376 Ideographic Tally Mark Five (identical to 正 U+6B63)
🄎 U+1F10E Circled Anticlockwise Arrow (as full-width)
🄏 U+1F10F Circled Dollar Sign With Overlaid Backslash (as full-width)
🄪 U+1F12A Tortoise Shell Bracketed Latin Capital Letter S (as full-width)
🄫 U+1F12B Circled Italic Latin Capital Letter C (as full-width)
🄬 U+1F12C Circled Italic Latin Capital Letter R (as full-width)
🄭 U+1F12D Circled Cd (as full-width)
🄮 U+1F12E Circled Wz (as full-width)
🄯 U+1F12F Copyleft Symbol (as full-width)
🅪 U+1F16A Raised MC Sign (based on existing raised TM sign U+2122)
🅫 U+1F16B Raised MD Sign
🅬 U+1F16C Raised MR Sign
🅭 U+1F16D Circled CC (as full-width)
🅮 U+1F16E Circled C With Overlaid Backslash (as full-width)
🅯 U+1F16F Circled Human Figure (as full-width)
🌍 U+1F30D Earth Globe Europe-Africa
🌎 U+1F30E Earth Globe Americas
🌏 U+1F30F Earth Globe Asia-Australia
🌐 U+1F310 Globe With Meridians
🌙 U+1F319 Crescent Moon (sized like other moon emojis)
🌟 U+1F31F Glowing Star (Emoji style)
🌭 U+1F32D Hot Dog
🌮 U+1F32E Taco
🌯 U+1F32F Burrito
🌰 U+1F330 Chestnut
🌲 U+1F332 Evergreen Tree
🌴 U+1F334 Palm Tree
🌵 U+1F335 Cactus
🌷 U+1F337 Tulip
🌹 U+1F339 Rose
🌺 U+1F33A Hibiscus
🌻 U+1F33B Sunflower
🌼 U+1F33C Blossom
🌽 U+1F33D Ear of Maize
🌾 U+1F33E Ear of Rice
🌿 U+1F33F Herb
🍁 U+1F341 Maple Leaf
🍂 U+1F342 Fallen Leaf
🍃 U+1F343 Leaf Fluttering In Wind
🍄 U+1F344 Mushroom
🍅 U+1F345 Tomato
🍆 U+1F346 Aubergine
🍇 U+1F347 Grapes
🍈 U+1F348 Melon
🍉 U+1F349 Watermelon
🍊 U+1F34A Tangerine
🍋 U+1F34B Lemon
🍌 U+1F34C Banana
🍍 U+1F34D Pineapple
🍐 U+1F350 Pear
🍑 U+1F351 Peach
🍒 U+1F352 Cherries
🍓 U+1F353 Strawberry
🍔 U+1F354 Hamburger
🍕 U+1F355 Slice of Pizza
🍖 U+1F356 Meat On Bone
🍗 U+1F357 Poultry Leg
🍘 U+1F358 Rice Cracker
🍙 U+1F359 Rice Ball
🍚 U+1F35A Cooked Rice
🍛 U+1F35B Curry and Rice
🍜 U+1F35C Steaming Bowl
🍝 U+1F35D Spaghetti
🍞 U+1F35E Bread
🍟 U+1F35F French Fries
🍠 U+1F360 Roasted Sweet Potato
🍡 U+1F361 Dango
🍢 U+1F362 Oden
🍣 U+1F363 Sushi
🍤 U+1F364 Fried Shrimp
🍥 U+1F365 Fish Cake with Swirl Design
🍦 U+1F366 Soft Ice Cream
🍧 U+1F367 Shaved Ice
🍨 U+1F368 Ice Cream
🍩 U+1F369 Doughnut
🍪 U+1F36A Cookie
🍫 U+1F36B Chocolate Bar
🍬 U+1F36C Candy
🍭 U+1F36D Lollipop
🍮 U+1F36E Custard
🍯 U+1F36F Honey Pot
🍰 U+1F370 Shortcake
🍱 U+1F371 Bento Box
🍲 U+1F372 Pot of Food
🍳 U+1F373 Cooking
🍴 U+1F374 Fork and Knife
🍵 U+1F375 Teacup Without Handle
🍶 U+1F376 Sake Bottle and Cup
🍷 U+1F377 Wine Glass
🍸 U+1F378 Cocktail Glass
🍺 U+1F37A Beer Mug
🍻 U+1F37B Clinking Beer Mugs
🍼 U+1F37C Baby Bottle
🍽 U+1F37D Fork and Knife with Plate (as full-width)
🍾 U+1F37E Bottle with Popping Cork
🍿 U+1F37F Popcorn
🎀 U+1F380 Ribbon
🎂 U+1F382 Birthday Cake
🎃 U+1F383 Jack-O-Lantern
🎄 U+1F384 Christmas Tree
🎅 U+1F385 Father Christmas
🎈 U+1F388 Balloon
🎉 U+1F389 Party Popper
🎊 U+1F38A Confetti Ball
🎋 U+1F38B Tanabata Tree
🎌 U+1F38C Crossed Flags
🎍 U+1F38D Pine Decoration
🎎 U+1F38E Japanese Dolls
🎏 U+1F38F Carp Streamer
🎐 U+1F390 Wind Chime
🎑 U+1F391 Moon Viewing Ceremony
🎒 U+1F392 School Satchel
🎓 U+1F393 Graduation Cap
🎩 U+1F3A9 Top Hat
🎪 U+1F3AA Circus Tent
🎬 U+1F3AC Clapper Board
🎯 U+1F3AF Direct Hit
🎰 U+1F3B0 Slot Machine
🎲 U+1F3B2 Game Die
🎳 U+1F3B3 Bowling
🎴 U+1F3B4 Flower Playing Cards
🎵 U+1F3B5 Musical Note
🎶 U+1F3B6 Multiple Musical Notes
🎹 U+1F3B9 Musical Keyboard
🎼 U+1F3BC Musical Score
🏀 U+1F3C0 Basketball and Hoop
🏅 U+1F3C5 Sports Medal
🏆 U+1F3C6 Trophy
🏈 U+1F3C8 American Football
🏉 U+1F3C9 Rugby Football
🏐 U+1F3D0 Volleyball
🏙 U+1F3D9 Cityscape (Emoji style)
🏚 U+1F3DA Derelict House Building (Emoji style)
🏛 U+1F3DB Classical Building (Emoji style)
🏠 U+1F3E0 House Building
🏡 U+1F3E1 House With Garden
🏢 U+1F3E2 Office Building
🏣 U+1F3E3 Japanese Post Office
🏤 U+1F3E4 European Post Office
🏥 U+1F3E5 Hospital
🏦 U+1F3E6 Bank
🏧 U+1F3E7 Automated Teller Machine
🏨 U+1F3E8 Hotel
🏩 U+1F3E9 Love Hotel
🏪 U+1F3EA Convenience Store
🏫 U+1F3EB School
🏬 U+1F3EC Department Store
🏭 U+1F3ED Factory
🏮 U+1F3EE Izakaya Lantern
🏯 U+1F3EF Japanese Castle
🏰 U+1F3F0 European Castle
🐓 U+1F413 Rooster
🐟 U+1F41F Fish
🐠 U+1F420 Tropical Fish
🐡 U+1F421 Blowfish
🐣 U+1F423 Hatching Chick
🐤 U+1F424 Baby Chick
🐥 U+1F425 Front-Facing Baby Chick
👀 U+1F440 Eyes
👁 U+1F441 Eye (Emoji style)
👄 U+1F444 Mouth
💈 U+1F488 Barber Pole
💊 U+1F48A Pill
💋 U+1F48B Kiss Mark
💍 U+1F48D Ring
💏 U+1F48F Kiss
💐 U+1F490 Bouquet
💑 U+1F491 Couple with Heart
💒 U+1F492 Wedding
💟 U+1F49F Heart Decoration
💦 U+1F4A6 Splashing Sweat Symbol
💧 U+1F4A7 Droplet
💱 U+1F4B1 Currency Exchange
💳 U+1F4B3 Credit Card
💴 U+1F4B4 Banknote With Yen Sign
💵 U+1F4B5 Banknote With Dollar Sign
💶 U+1F4B6 Banknote With Euro Sign
💷 U+1F4B7 Banknote With Pound Sign
💸 U+1F4B8 Money with Wings
💹 U+1F4B9 Chart with Upwards Trend and Yen Sign
📈 U+1F4C8 Chart with Upwards Trend
📉 U+1F4C9 Chart with Downwards Trend
📊 U+1F4CA Bar Chart
📍 U+1F4CD Round Pushpin
📎 U+1F4CE Paperclip
📟 U+1F4DF Pager
📱 U+1F4F1 Mobile Phone
📲 U+1F4F2 Mobile Phone With Rightwards Arrow At Left
📹 U+1F4F9 Video Camera
📺 U+1F4FA Television
📻 U+1F4FB Radio
📼 U+1F4FC Videocassette
🔌 U+1F50C Electric Plug
🔍 U+1F50D Left-Pointing Magnifying Glass
🔎 U+1F50E Right-Pointing Magnifying Glass
🔏 U+1F50F Lock with Ink Pen
🔦 U+1F526 Electric Torch
🔨 U+1F528 Hammer
🔩 U+1F529 Nut and Bolt
🔰 U+1F530 Japanese Symbol For Beginner
🔱 U+1F531 Trident Emblem
🕹 U+1F579 Joystick (Emoji style)
🗄 U+1F5C4 File Cabinet (Emoji style)
🗻 U+1F5FB Mount Fuji
🗼 U+1F5FC Tokyo Tower
🗾 U+1F5FE Silhouette of Japan
😍 U+1F60D Smiling Face with Heart-Shaped Eyes
🚥 U+1F6A5 Horizontal Traffic Light
🚦 U+1F6A6 Vertical Traffic Light
🚧 U+1F6A7 Construction Sign
🚨 U+1F6A8 Police Cars Revolving Light
🚩 U+1F6A9 Triangular Flag On Post
🚪 U+1F6AA Door
🚫 U+1F6AB No Entry Sign
🛑 U+1F6D1 Octagonal Sign
🛕 U+1F6D5 Hindu Temple
🛖 U+1F6D6 Hut
🛫 U+1F6EB Airplane Departure
🛬 U+1F6EC Airplane Arriving
🤩 U+1F929 Grinning Face with Star Eyes
🥀 U+1F940 Wilted Flower
🥇 U+1F947 First Place Medal
🥈 U+1F948 Second Place Medal
🥉 U+1F949 Third Place Medal
🥎 U+1F94E Softball
🥚 U+1F95A Egg
🥛 U+1F95B Glass of Milk
🥤 U+1F964 Cup with Straw
🥮 U+1F96E Moon Cake
🥯 U+1F96F Bagel
🥷 U+1F977 Ninja
🦋 U+1F98B Butterfly
🧉 U+1F9C9 Mate Drink
🧊 U+1F9CA Ice Cube
🧋 U+1F9CB Bubble Tea
🧨 U+1F9E8 Firecracker
🧩 U+1F9E9 Jigsaw Puzzle Piece
🧭 U+1F9ED Compass
🧱 U+1F9F1 Brick
🧲 U+1F9F2 Magnet
🧵 U+1F9F5 Spool of Thread
🧶 U+1F9F6 Ball of Yarn
🧷 U+1F9F7 Safety Pin
🧸 U+1F9F8 Teddy Bear
🧺 U+1F9FA Basket
🧻 U+1F9FB Roll of Paper
🧼 U+1F9FC Bar of Soap
🧽 U+1F9FD Sponge
🩸 U+1FA78 Drop of Blood
🩹 U+1FA79 Adhesive Bandage
🪙 U+1FA99 Coin
🪚 U+1FA9A Carpentry Saw
🪛 U+1FA9B Screwdriver
🪠 U+1FAA0 Plunger
🪪 U+1FAAA Identification Card
🫕 U+1FAD5 Fondue
🫖 U+1FAD6 Teapot
🫗 U+1FAD7 Pouring Liquid
🫙 U+1FAD9 Jar
🫦 U+1FAE6 Biting Lip
🫧 U+1FAE7 Bubbles

PhMajerus committed:
Fix box drawings diagonals

This changes the 3 box drawings diagonals to create staight lines (as per issue #160).

Updated:
╱ U+2571 Box Drawings Light Diagonal Upper Right to Lower Left
╲ U+2572 Box Drawings Light Diagonal Upper Left to Lower Right
╳ U+2573 Box Drawings Light Diagonal Cross

Note the extra diagonals that are part of the Symbols for Legacy Computing block were already using this style.

PhMajerus committed:
Fix ~ symbols consistency

This changes the 2 old symbols to make them consistent with the other tilde symbols, as per issue #154.

Updated:
≈ `U+2248` Almost Equal To
≉ `U+2249` Not Almost Equal To

PhMajerus committed:
Adjustments to block elements to unify grid

Uses a unified grid system for block elements U+2580 to U+259F, U+1FB70 to U+1FB7B, and U+1FB82 to U+1FB8B, ensuring consistent pseudo-pixels positions.

PhMajerus committed:
Adjusted MouseText and a few other symbols for legacy computing

Adjusted U+1FB7C to U+1FB81, U+1FBB5 to U+1FBB8, U+1FBBC, U+1CC8E and U+1CC8F to fit the new unified grid.

PhMajerus committed:
Adjusted one quarter and three quarters block elements
Adjusted U+1CEA5 to U+1CEA7 and U+1CEAC to U+1CEAE to fit the new unified grid

bluetoad07 commented:
love the new kanji!! here's a couple changes i'd make though for consistency with the existing ones and just general readability
<img width="105" height="43" alt="phm_kanji-export" src="https://github.com/user-attachments/assets/4c2e3338-115d-465d-aaa1-e1265082ae82" />

PhMajerus commented:
@bluetoad07 ok for almost all your changes except for 忍. The top must be 刃, with the stroke crossing the 刀.
I'll revisit them and apply your changes, but for this one we need to find a way to keep the strokes correct, maybe like this?
<img width="14" height="15" alt="image" src="https://github.com/user-attachments/assets/a81a0ce5-e5f9-4dfb-a477-74f5a52572f4" />

bluetoad07 commented:
@PhMajerus yeah that looks good to me! that radical can be written both ways but i think you're right, it's better with the cross

PhMajerus commented:
> @PhMajerus yeah that looks good to me! that radical can be written both ways but i think you're right, it's better with the cross

Interesting, I've never seen it with the stroke not crossing but you're right, and the Japanese font on Android uses that design.
I believe all the Microsoft Japanese fonts use the crossing design, ever since MS-DOS/V, and so does Shinonome. And so does Sega for their Shinobi games titles, so I've never encountered the other design.
<img width="794" height="278" alt="image" src="https://github.com/user-attachments/assets/8a8c1efc-b75e-4700-8a48-58e7a340d178" />

BTW, if it can help you with the kanjis design, I finally got around to build Shinonome 12pts as a figlet font using octants, so it's easy to have a look at the designs used in both DOS/V and Shinonome from the terminal.

PhMajerus committed:
Kanjis adjustments following bluetoad07's review

PhMajerus commented:
@bluetoad07 I made the changes and only adjusted slightly 忍 and 舞 from your suggestions (in blue below):
<img width="105" height="43" alt="image" src="https://github.com/user-attachments/assets/e1d92af1-4084-4e11-b3e3-6dea39c68822" />

This seems more readable and consistent with existing kanjis, what do you think?

bluetoad07 commented:
@PhMajerus looks great!! thank you!

PhMajerus commented:
@bluetoad07 
It would be nice to add 剣者龍, but I'm not sure how to fit them nicely in 11×11 and which design schemes you want for those.
Some ideas:
<img width="40" height="15" alt="image" src="https://github.com/user-attachments/assets/6c7a0d62-79e9-4f90-997f-b0e52554c05b" />

PhMajerus committed:
Added 3 new kanjis in collaboration with @bluetoad07

Added:
剣 U+5263 CJK Unified Ideograph-5263
者 U+8005 CJK Unified Ideograph-8005
龍 U+9F8D CJK Unified Ideograph-9F8D

PhMajerus commented:
Great, I added them with your design.
This is it for now, I'll be very busy for a month or two and probably won't be able to contribute anything during that time.

PhMajerus committed:
Bells
Added:
🔔 U+1F514 Bell (as Emoji)
🔕 U+1F515
🯺 U+1FBFA Alarm Bell Symbol (Symbols for Legacy Computing Unicode 17.0)

(Note this does not add the Unicode 17 additions to the Symbols for Legacy Computing Supplement block.)

PhMajerus committed:
Consistency adjustment
Updated 🔕 U+1F515 Bell with Cancellation Stroke to make it more similar to existing U+F009B.

PhMajerus committed:
Added 5 more emojis
Added:
🕾 U+1F57E White Touchtone Telephone
🕿 U+1F57F Black Touchtone Telephone
🖄 U+1F584 Envelope with Lightning
🖅 U+1F585 Flying Envelope
🖆 U+1F586 Pen Over Stamped Envelope

PhMajerus committed:
Added 33 more emojis
Added:
🌠 U+1F320 Shooting Star
🌶 U+1F336 Hot Pepper
💢 U+1F4A2 Anger Symbol
💨 U+1F4A8 Dash Symbol
📛 U+1F4DB Name Badge
🥁 U+1F941 Drum with Drumsticks
🥐 U+1F950 Croissant
🥑 U+1F951 Avocado
🥒 U+1F952 Cucumber
🥔 U+1F954 Potato
🥕 U+1F955 Carrot
🥖 U+1F956 Baguette Bread
🥙 U+1F959 Stuffed Flatbread
🥜 U+1F95C Peanuts
🥝 U+1F95D Kiwifruit
🥞 U+1F95E Pancakes
🥟 U+1F95F Dumpling
🥠 U+1F960 Fortune Cookie
🪎 U+1FA8E Treasure Chest
🪏 U+1FA8F Shovel
🪾 U+1FABE Leafless Tree
🫀 U+1FAC0 Anatomical Heart
🫁 U+1FAC1 Lungs
🫆 U+1FAC6 Fingerprint
🫐 U+1FAD0 Blueberries
🫑 U+1FAD1 Bell Pepper
🫒 U+1FAD2 Olive
🫓 U+1FAD3 Flatbread
🫔 U+1FAD4 Tamale
🫘 U+1FAD8 Beans
🫚 U+1FADA Ginger
🫛 U+1FADB Pea Pod
🫜 U+1FADC Root Vegetable

the-moonwitch commented:
@Eejit43 yeah, I noticed - I've been meaning to move the builds over to `uv` and clean up the build code in general, 
just wanted to get all the tricky merges sorted out ASAP to unblock people. I'll look at it shortly, hopefully.
